implementations/c/ockam/channel/tests/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+
+if(NOT BUILD_TESTING)
+    return()
+endif()
+
+find_package(cmocka QUIET)
+if(NOT cmocka_FOUND)
+    return()
+endif()
+
+# ---
+# okcam_channel_tests
+# ---
+add_executable(okcam_channel_tests
+        channel_test.h
+        channel_test.c
+        initiator.c
+        responder.c)
+
+target_link_libraries(
+    okcam_channel_tests
+    PUBLIC
+        ockam_vault_default
```

Comment:
Should be replaced with namespaced alias

Reply:
yes.